bin/src/server/main.rs

```diff
@@ -28,35 +21,25 @@ fn main() -> Result<(), errors::Error> {
     let listener = exec.run_until(async { TcpListener::bind(&addr).await })?;
     println!("{}", listener.local_addr()?);
 
-    let connection_string = env::var("DATABASE_URL").unwrap_or_else(|_| "memory://".to_string());
-
-    if connection_string.starts_with("rocksdb://") {
-        let path = &connection_string[10..connection_string.len()];
-
-        let max_open_files_str = env::var("ROCKSDB_MAX_OPEN_FILES").unwrap_or_else(|_| "512".to_string());
-        let max_open_files = max_open_files_str.parse::<i32>().expect(
-            "Could not parse environment variable `ROCKSDB_MAX_OPEN_FILES`: must be an \
-             i32",
-        );
-
-        let bulk_load_optimized = env::var("ROCKSDB_BULK_LOAD_OPTIMIZED").unwrap_or_else(|_| "".to_string()) == "true";
-
-        let datastore = indradb::RocksdbDatastore::new(path, Some(max_open_files), bulk_load_optimized)
-            .expect("Expected to be able to create the RocksDB datastore");
+    if args.database_url.starts_with("rocksdb://") {
```

Comment:
Instead of parsing the database URL here, you can use `subcommand_matches`: https://docs.rs/clap/2.33.3/clap/struct.ArgMatches.html#method.subcommand_matches